[2021-08-04 11:46:58,952] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: user_processing.processing_user 2021-08-03T00:00:00+00:00 [queued]>
[2021-08-04 11:46:58,964] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: user_processing.processing_user 2021-08-03T00:00:00+00:00 [queued]>
[2021-08-04 11:46:58,965] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 11:46:58,965] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-08-04 11:46:58,967] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 11:46:59,013] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): processing_user> on 2021-08-03T00:00:00+00:00
[2021-08-04 11:46:59,017] {standard_task_runner.py:52} INFO - Started process 16047 to run task
[2021-08-04 11:46:59,029] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'user_processing', 'processing_user', '2021-08-03T00:00:00+00:00', '--job-id', '31', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpwmiqqxpt', '--error-file', '/tmp/tmp4yok2bzf']
[2021-08-04 11:46:59,034] {standard_task_runner.py:77} INFO - Job 31: Subtask processing_user
[2021-08-04 11:46:59,098] {logging_mixin.py:104} INFO - Running <TaskInstance: user_processing.processing_user 2021-08-03T00:00:00+00:00 [running]> on host airflowvm
[2021-08-04 11:46:59,147] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=processing_user
AIRFLOW_CTX_EXECUTION_DATE=2021-08-03T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-03T00:00:00+00:00
[2021-08-04 11:46:59,157] {python.py:151} INFO - Done. Returned value was: None
[2021-08-04 11:46:59,161] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=processing_user, execution_date=20210803T000000, start_date=20210804T114658, end_date=20210804T114659
[2021-08-04 11:46:59,197] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-04 11:46:59,200] {local_task_job.py:151} INFO - Task exited with return code 0
